--- src/notify.c
+++ src/notify.c
@@ -614,10 +614,11 @@
                                   item->icon_path ? item->icon_path : "",
                                   item->summary ? item->summary : "",
                                   item->body ? item->body : "",
-                                  item->desktop_entry ? item->desktop_entry : "");
+                                  item->desktop_entry ? item->desktop_entry : "",
+                                  item->value);
         }
         
-        g_dbus_method_invocation_return_value(invocation, g_variant_new("(a(usssss))", builder));
+        g_dbus_method_invocation_return_value(invocation, g_variant_new("(a(ussssssi))", builder));
         g_variant_builder_unref(builder);
     }
     // استدعاء الحدث الافتراضي (للكونترول سنتر)
