--- src/main.c
+++ src/main.c
@@ -1360,7 +1360,20 @@
 		NotificationData *n = history->data;
 		state->latest_notif_app = g_strdup(n->app_name);
 		state->latest_notif_summary = g_strdup(n->summary);
-		state->latest_notif_body = g_strdup(n->body);
+		
+		if (n->body) {
+			GRegex *regex = g_regex_new("<a[^>]*>|</a>", 0, 0, NULL);
+			state->latest_notif_body = g_regex_replace_literal(regex, n->body, -1, 0, "", 0, NULL);
+			g_regex_unref(regex);
+			
+			GError *err = NULL;
+			if (!pango_parse_markup(state->latest_notif_body, -1, 0, NULL, NULL, NULL, &err)) {
+				gchar *escaped = g_markup_escape_text(state->latest_notif_body, -1);
+				g_free(state->latest_notif_body);
+				state->latest_notif_body = escaped;
+				g_error_free(err);
+			}
+		}
 		state->latest_notif_value = n->value;
 		
 		if (n->icon_path) {
